main.nf

```diff
@@ -61,6 +61,9 @@ workflow {
     NFCORE_SMRNASEQ (
         Channel.of(file(params.input, checkIfExists: true)),
         PIPELINE_INITIALISATION.out.samplesheet,
+        params.fasta,
+        params.mirtrace_species,
+        params.bowtie_index,
```

Comment:
I'll leave it as it is, as we discussed, since there doesn't seem to be a standard for how to set up the new template, we can always change it later if we have a reason to do so